tests/Doctrine/Tests/Common/Cache/CacheTest.php

```diff
@@ -70,9 +70,9 @@ public function testFetchMultiWillFilterNonRequestedKeys()
             ->expects($this->once())
             ->method('doFetchMultiple')
             ->will($this->returnValue(array(
-                '[foo][]' => 'bar',
-                '[bar][]' => 'baz',
-                '[baz][]' => 'tab',
+                '[foo][1]' => 'bar',
```

Comment:
I just had to change that because `doFetch` wrongly returned null instead of false in the mocked tests.